linux/helper_suse.sh

```diff
@@ -1,16 +1,9 @@
 # Adds the NR repo
 add_repo() {
-    if [ "$STAGING_REPO" = "true" ]; then
-        repo="http://nr-downloads-ohai-staging.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/zypp/sles/12.4/x86_64/newrelic-infra.repo"
-    else
-        repo="http://nr-downloads-main.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/zypp/sles/12.4/x86_64/newrelic-infra.repo"
-    fi
-
     zypper -n install wget gnupg
-    wget -nv -O /etc/zypp/repos.d/newrelic-infra.repo $repo
-    # prod .repo file points to the cache url, we replace it to point to the bucket url
-    rs='s|https://download.newrelic.com|http://nr-downloads-main.s3-website-us-east-1.amazonaws.com|'
-    sed -i "$rs" /etc/zypp/repos.d/newrelic-infra.repo
+
+    [ "$STAGING_REPO" = "true" ] && env="-staging" || env="" ;
```

Comment:
I'd suggest to use the longer `if []; then` syntax, it ends up being more readable.
```suggestion
    env=""
    if [ "$STAGING_REPO" = "true" ]; then
        env="-staging"
    fi
```